### ISSUE TITLE
Issues with this concept

### ISSUE THREAD

**Issue by faddat** [Thyb0rg/cosmos-validators-league#1]
* allows the abdication of thought by key organizations
* "requires" things whose use cannot be confirmed (horcrux, tmkms) -- and which are outside the chains themselves. 
* promotes its own use cosmos-wide, promoting centralization
* if implemented as described, will result in the usage of the same list by several key orgs on the cosmos hub, making this list, not gov the ultimate power broker

This should not be a list, it should be policies. 

If we can't confirm a policy that it can't be a policy: we can find that would not be good.

Please let me know if you are receptive to a pull request that solves day is issues. Otherwise, it is likely necessary for me to leave the discord server and label this effort for what it is, and attempt to literally control cosmos from the ground up.

If we are interested in modifying the document, dramatically, away from a list, to a set of policies, which we could then hold organizations accountable to, that's great with me. But otherwise I don't see that's going very far and I do see it harming the networks in cosmos.

**Comment by Thyb0rg** [comment]
Yes, open to updates to the document

There seems to be some misunderstanding wrt to our intentions on policy
setting however

A similar conversation has been going on on Discord, so I suggest you have
a look at that first

Would also recommend to wait for the upcoming vote to gauge the support for
this project

 If it turns out there's not enough community validators interested in
seeing it come through, it might get cancelled altogether.

**Comment by AFDudley** [comment]
I'm sorry, this is a horrible idea. Please just implement "one validator, one vote" and be done with it. You're doing way too much over here. Way too much.

**Comment by AFDudley** [comment]
Do you have a link to the discord this is being discussed in?

**Comment by Thyb0rg** [comment]
Please message me on Twitter for the invitation link @Thyborg_

**Comment by adizere** [comment]
>  If it turns out there's not enough community validators interested in
seeing it come through, it might get cancelled altogether.

Interesting.

I'm still curious: What is the relation between the ranking list being compiled as part of the effort in this repo and the ICF [delegation issue discussed in this page](https://thoracic-boron-678.notion.site/ICF-Delegation-Report-b4b8c62295564990800fb1226d585e4b)?

If I'm understanding correctly, there are two distinct problems I heard about recently:

1. Who should the ICF delegate to? This is the issue raised in the page above.
2. How to deal with the "concentration of economic and voting power"? This is the issue raised in the present repo.

There _may_ be an overlap between the two problems, but it's not clear to me the solution is the same. Some members of the community are concerned that using the solution to problem (2) to fix problem (1) is not a good idea.

**Comment by Thyb0rg** [comment]
Hey Adi,

Sorry for the late reply here.

There's no direct relation, although I do think gathering & providing good
validators data could help inform with delegation programs

We also moved away from establishing a "canonical" ranking to instead
enable any third-party to build theirs.

**Comment by serejandmyself** [comment]
Seems the issue is not the validators, but the approach taken to them by everyone. Seems that the first step is to change the way we view blockchains as such. This will help to understand the role of validators

**Comment by JSONbored** [comment]
> Seems the issue is not the validators, but the approach taken to them by everyone. Seems that the first step is to change the way we view blockchains as such. This will help to understand the role of validators

Agreed. The aim here is to truly just offer up freely accessible information to the community at large about validators across all chains, in a way that's easy to digest and does as best as possible to eliminate any "builder bias" / favoritism towards certain validators over others.

We need more sources of info that are easy for the community to understand and take into consideration when choosing validators to stake with - hopefully, this can be one of them (although I'd LOVE to see many more variants / UIs pop up from other teams). 

We've had a lot of productive conversations in the Discord, and I feel that we're on a pretty good track now - it may not be *perfect*, but it's a start - at the end of the day, I think the majority of teams truly care about decentralization and keeping our ecosystem healthy & secure. 

Hopefully, the data provided by tools like these help us accomplish these goals, allowing the community to make more informed decisions when staking.

**Comment by mohammedpatla** [comment]
Would love to contribute on this as well!
